builders.py

```diff
@@ -231,7 +231,7 @@ def rtc_action(target, source, env):
 
     args.append(source_dir)
     jobs = env.GetOption("num_jobs")
-    rtc_env = Environment()
+    rtc_env = Environment(ENV = os.environ)
```

Comment:
This is the only required change to detect cmake since this is the only part of the build system where we use cmake. The other two changes should be removed before this can be merged

Reply:
I think we could actually do even better and use `SCons.Util.WhereIs` instead, I'll give it a try.

Reply:
ok I updated the PR to just do this and it appears to work on my OSX setup. I haven't tested on windows tho